### PR TITLE
refactor code alter collumn name: value to word

### PR DESCRIPTION
**Pull request by Danilosimoes**

### CODE REVIEW

File: src/it/java/com/uvocab/api/controller/VocabularyControllerTest.java [hunk 1]
```diff
@@ -10,20 +10,21 @@
...
 @ActiveProfiles({"test"})
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
-class WordControllerTest {
+class VocabularyControllerTest {
   @Autowired private TestRestTemplate restTemplate;
 
   @Test
   void shouldpostWord() {
-    var value = Word.newBuilder().setId(1).setValue("Home").build();
+    var word = Vocabulary.newBuilder().setId(1).setWord("Home").build();
```
Comment (robsonoduarte) [inline src/it/java/com/uvocab/api/controller/VocabularyControllerTest.java:22]: @Danilosimoes why no, `var vocabulary..

**Review by robsonoduarte — COMMENTED** [review]
a simple question

File: src/it/java/com/uvocab/api/controller/VocabularyControllerTest.java (hunk 1 above)
Comment (Danilosimoes) [reply src/it/java/com/uvocab/api/controller/VocabularyControllerTest.java:22]: Cause we're returning a word from vocabulary.

Comment (robsonoduarte) [reply src/it/java/com/uvocab/api/controller/VocabularyControllerTest.java:22]: return a word or build a Vocabulary ?

Comment (Danilosimoes) [reply src/it/java/com/uvocab/api/controller/VocabularyControllerTest.java:22]: You are correct!

**Review by robsonoduarte — APPROVED** [review]
:rocket:

**Pull request merged**